examples/lua/bashreadline.lua

```diff
@@ -0,0 +1,30 @@
+--[[
+Copyright 2016 GitHub, Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+]]
+local ffi = require("ffi")
+
+return function(BPF)
+  local b = BPF:new{src_file="bashreadline.c", debug=true}
```

Comment:
Two things:
* We've found that inlining the C source contents (which you do in memleak.lua) is more portable. I would suggest adding relative path support or inlining everything, since a user needs to be in the same directory to be able to find bashreadline.c here.
* debug is not a bool, but rather a set of flags. See #456